src/main/java/teamrtg/rtg/core/world/ChunkProviderRTG.java

```diff
@@ -748,7 +605,7 @@ private void doPopulate(int x, int z) {
                     RealisticBiomeGenerator.forBiome(realisticBiome.getBiome()).rDecorate(rtgWorld, rand, worldX, worldZ, borderNoise[bn], river, hasPlacedVillageBlocks);
                 } else {
                     try {
-                        realisticBiome.getBiome().decorate(this.world, rand, worldCoords);
+                        realisticBiome.getBiome().decorate(this.world, rand, blockpos);
                     } catch (Exception e) {
                         RealisticBiomeGenerator.forBiome(realisticBiome.getBiome()).rDecorate(rtgWorld, rand, worldX, worldZ, borderNoise[bn], river, hasPlacedVillageBlocks);
```

Comment:
There are plently of long lines in this class that can be chopped down for better readability.